website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -118,12 +118,37 @@ set on the _BQ test_ project.
 />
 
 
+### Manual assignment
+
+<Changelog>
+
+- New in version 1.1.23 (March, 2021)
+
+</Changelog>
+
+dbt Cloud administrators can manually assign users to groups independently of
+IdP attributes. If a dbt Cloud group is configured _without_ any
+SSO Mappings, then the group will be _unmanaged_ and dbt Cloud will not adjust
+group membership automatically when users log into dbt Cloud via an identity
+provider. This behavior may be desirable for teams that have connected an identity
+provider, but have not yet configured SSO Mappings between dbt Cloud and the
+IdP.
+
+Note: If an SSO Mapping is added to an _unmanaged_ group, then it will become
```

Comment:
This is not my area of expertise so take this with a grain of salt. 😄 

Do we need `Note` here? 🤔 This is desired normal behavior and not an exclusive edge case that needs user attention. 

```suggestion
If an SSO Mapping is added to an _unmanaged_ group, then it will become
```

Reply:
you're so right - updating :)